src/Curators/Publisher.php

```diff
@@ -20,24 +14,19 @@ class Publisher
     private function __construct(string $name)
     {
         if (!self::isValid($name)) {
-            throw new InvalidArgumentException('Unknown publisher: ' . $name);
+            throw new InvalidArgumentException('Invalid publisher: ' . $name);
         }
         $this->name = $name;
     }
 
-    public static function bruzz(): self
-    {
-        return new self(self::BRUZZ);
-    }
-
     public static function fromName(string $name): self
```

Comment:
Isn't the value object itself technically a (publisher) name? So `fromName()` does not seem very clear. `fromString()` seems better no? But there's also not much added value over using the constructor in that case so maybe we don't even need this method anymore?

Reply:
I could rename it `PublisherName` and just use the constructor?